testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -4777,6 +4777,64 @@ QUnit.module('keyboard navigation', {
         assert.ok($cancelButton.hasClass('dx-state-focused'), 'cancel button is focused');
     });
 
+    QUnit.testInActiveWindow('Hour input has focused state if tab key was pressed when dropDownOptions: { toolbarItems: [] }', function(assert) {
+        if(devices.real().deviceType !== 'desktop') {
+            assert.ok(true, 'test does not actual for mobile devices');
+            return;
+        }
+
+        const $dateBox = $('#dateBox').dxDateBox({
+            type: 'datetime',
+            dropDownOptions: { toolbarItems: [] },
+            opened: true,
+            focusStateEnabled: true,
+        });
+
+        const instance = $dateBox.dxDateBox('instance');
+        const $input = $dateBox.find(`.${TEXTEDITOR_INPUT_CLASS}`);
+
+        const keyboard = keyboardMock($input);
+
+        keyboard.keyDown('tab');
+
+        const $hourBox = $(instance._strategy._timeView._hourBox.$element());
+        assert.ok($hourBox.hasClass(STATE_FOCUSED_CLASS), 'tab set focus to first input in overlay');
+
+        const $hourInput = instance._strategy._timeView._hourBox._input();
```

Comment:
May be it will better to get this $element using selector instead of private method using?

Reply:
Fixed in [d926b33defc91a284d9191741af5b9c9f64cf3fd]